tests/utils/test_exporter.py

```diff
@@ -497,7 +497,7 @@ def test_exporter_can_export_requirements_txt_with_git_packages(tmp_dir, poetry)
         content = f.read()
 
     expected = """\
-foo @ git+https://github.com/foo/foo.git@123456
+-e foo @ git+https://github.com/foo/foo.git@123456
```

Comment:
Hmm, these are not marked as `develop` in the lock data above. This should not, be `-e` prefixed. I think there is a bug some place else. I wonder if https://github.com/python-poetry/poetry-core/pull/98 is related?

Reply:
Ah, I felt like I saw a PR for this. I can wait for that to land and then rebase my change here.

Reply:
Would be great if you can test with that branch locally atleast to confirm this is the case. Else we will have to dig into it seperately.

Reply:
Verified that against that branch the tests don't need updating.